storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/KvStoreAccessor.java

```diff
@@ -82,8 +94,31 @@
 
     <T> void put(KvStoreVariable<T> variable, T value);
 
+    /**
+     * Write raw bytes to a specified variable WARNING: should only be used to migrate data between
+     * database instances
+     *
+     * @param variable
+     * @param value
+     * @param <T>
```

Comment:
We should either add descriptions for these or delete them to avoid javadoc warnings.